OperacoesListaSequencial.cpp

```diff
@@ -0,0 +1,459 @@
+#include <stdlib.h>
+#include <iostream>
+#include <new>
+#include <string>
+
+using namespace std;  // para utilizar cout
+
+// Struct para pessoa
+struct pessoa {
+  string nome;
+  int rg;
+};
+
+// Limpa a tela
+void limpatela() {
+  system("CLS");
+}
+
+// Imprime a lista Sequencial
+void imprimeSequencial(pessoa* ponteiroSequencial,
+                       int tamanhoDaListaSequencial) {
+  // Cabecaalho da Lista
+  cout << "\nLista: \n";
```

Comment:
this should be changed to the printing method that will be used by the code, something like Serial.print